lib/streamer.js

```diff
@@ -15,12 +15,12 @@ const StartPlayback = require('./StartPlayback.js').StartPlayback;
 const Hello = require('./Hello.js').Hello;
 
 class NexusStreamer extends EventEmitter {
-  constructor(host, cameraUUID, sessionToken, useOMX) {
+  constructor(host, cameraUUID, sessionToken, config) {
     super();
     let self = this;
     self.isStreaming = false;
     self.authorized = false;
-    self.useOMX = useOMX;
+    self.ffmpegCodec = config.ffmpegCodec;
```

Comment:
Here you are accessing `config.ffmpegCodec` but in `NestCam.configureWithHAP` we are doing the fallback on `NestCam.ffmpegCodec`, maybe either have `NestCam.configureWithHAP` override config directly, or pass the `ffmpegCodec` as part of the `NexusStreamer` constructor?